ui/packages/components/src/CodeSearch/CodeSearch.tsx

```diff
@@ -370,8 +373,15 @@ export default function CodeSearch({
     }, VALIDATION_DELAY);
   };
 
+  const expressionError = searchError || editorValidationError;
```

Comment:
showing `editorValidationError` in the `Alert` makes the UI slightly more consistent, but the error messages from the frontend and backend validation are currently inconsistent

Reply:
I decided to flip the order here since currently the frontend validation error is more informative than the backend one